content/blog/2018/02/2018-02-27-browser-usage.adoc

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: "Browser Market Share on Jenkins.io"
+tags:
+- community
+- stats
+- browser
+author: olblak
+---
+
+= Browser Market Share on Jenkins.io
+
+I recently had a look to the google analytics account used for the main jenkins website. +
+It was interesting to discover which browsers are used by Jenkins visitors and how this compares with other websites. +
+I decided to analyse one year of data from January 2017 to January 2018. +
+I selected http://gs.statcounter.com/about[statcounter] as an external data source in order to compare results from jenkins.io.
+
+
+== Jenkins Visitors
+
+Let's talk about numbers: +
+During that period, 3,496,245 users across the world visited this website. +
+This is a growth of 152.69% visitors compared to the year before. +
+92.28% of them used a desktop, 6.63% were on mobile and only 1.10% on a tablet.
+
+[cols="a,a"]
+|===
+|
+|
+|
+.Jenkins Visitors' Browser Usage
+[cols="1,10,1",width="100%"]
+!===
+!#! Browser! %
+
+! 1. 
+! Chrome
+! 69.82%
+
+! 2.
+! Firefox
+! 14.75%
+
+! 3.
+! Safari 
+! 6.44%
+
+! 4.
+! IE
+! 5.84%
+
+! 5. 
+! Edge
+! 1.59%
+
+! 6.
+! Opera
+! 0.55%
+
+! 7. 
+! Android Webview
+! 0.23%
+
+! 8.
+! Safari (in-app)
+! 0.23%
+
+! 9.
+! Ya Browser
+! 0.11%
+
+! 10.
+! UC Browser
+! 0.10%
+!===
+|
+.Jenkins Visitors' Browser Usage
+image:/images/post-images/2018-02-29/jenkins_browser_2017-2018.png[width="300",align=center]
+
+We clearly see that three browsers lead this market share with a total of 91%.
```

Comment:
Maybe we should avoid "market share" in Jenkins-specific sections

Reply:
I can change that